tests/unit/components/pacbio/PacbioWell.spec.js

```diff
@@ -110,6 +122,16 @@ describe('Well.vue', () => {
       storeWell.sequencing_mode = ""
       storeWell.on_plate_loading_concentration = ""
       storeWell.insert_size = ""
+
+      wrapper = mount(Well, { 
+        localVue, 
+        store, 
+        propsData: props,
+        stubs: {
+          WellModal: true
+        }
+      })
+  
```

Comment:
How come the re-mounts are required here 3 times?

Reply:
You can no longer set props data after it has been mounted. It is not reactive. I think it is part of the new way.